### ISSUE TITLE
Document what Marvin is & isn't

### ISSUE THREAD

**Issue by omnomicon** [sethtrain/marvin#27]
* Discuss on this ticket
* Should be added to README once complete
* Should be an easy-to-understand Occam's Razor-type rubric for deciding if a thing fits in Marvin

**Comment by omnomicon** [comment]
For me, I've been taking the approach of trying to eliminate the things which are easy to do face-to-face, but hard or impossible virtually (via hangouts, skype, etc.).

Dice-rolling is for me the perfect example. In a face-to-face game, you just get out the dice and roll them. Virtually, it would be disappointing for someone to roll privately, everyone hears the dice bouncing across the desk, then hear "Hey, I totes rolled a natural 20. Again!"

What are other things Marvin can help with? Are there any abstract principles we can pull out of the list of things Marvin needs to implement?

**Comment by sethtrain** [comment]
Yep, I'm in line with the "eliminate the things which are easy to do face-to-face but hard or impossible virtually".  I tend to lean towards "because it is virtual are there easy wins that will help?".

So my thoughts from a GM perspective...

If I can have certain tools at my disposal that will help keep me organized then it seems like an easy thing to do, most of these things revolve around the narrative for the game in progress.  As a part of that narrative I have the responsibility to tell the story in a manner that gives every possible opportunity for the player imaginations to "run freely".  With this responsibility in mind, I wonder what are the things that could help me.  Playing sounds, showing pictures and displaying (with as much detail as I desire and feel the need to use) the current state of things, whether that be in map form for visual association of NPCs or to help with the story.

Given these things can provide GM help with what I feel like would be little effort it seems like an easy win opening up more opportunity to get _all_ the people using the system to have a better time.

It would also be nice to have an "electronic record" for easy reuse, historical reminiscing and reflection.

The main goal is support for better story telling, can this all be done with pencil and paper, sure.  Does it provide value with little effort, I think so.
